app/navigation/Navigation.scala

```diff
@@ -84,6 +80,18 @@ class Navigation extends Navigator {
         }
     }
 
+  private def revisedUnloadingProcedureConditionsYesNoNavigation(ua: UserAnswers, mode: Mode): Option[Call] =
+    mode match {
+      case NormalMode => Some(Call("Get", "#")) // Update with Normal mode Navigation
```

Comment:
Is this still outstanding or will this be covered in Kara's branch? Do these two branches need combining into one testable feature branch?